src/backgroundMessaging.ts

```diff
@@ -0,0 +1,8 @@
+import firebase from "react-native-firebase";
+import { RemoteMessage } from "react-native-firebase/messaging";
+
+// Must return within 60 seconds.
+// See: https://rnfirebase.io/docs/v4.3.x/messaging/receiving-messages#1)-Add-Service-to-Android-Manifest
+export default async (message: RemoteMessage) => {
+  return Promise.resolve();
```

Comment:
Why are we not handling the message? looks like we're just throwing it away

Reply:
If there's a good reason for this, then please comment

Reply:
Removing these for now until we actually need them.